client/src/app/shared/components/jitsi/jitsi.component.ts

```diff
@@ -227,6 +263,9 @@ export class JitsiComponent extends BaseViewComponent implements OnInit, OnDestr
 
     private async setUp(): Promise<void> {
         this.subscriptions.push(
+            // if the operators users has changes, check if we have to start the animation
+            this.operator.getUserObservable().subscribe(() => this.toggleButtonVisibleState()),
```

Comment:
If you watch for all operator changes, `canManageSpeaker` can also be evaluated here and just saved into a variable.